CMakeLists.txt

```diff
@@ -161,14 +161,25 @@ if(CMAKE_SYSTEM_NAME STREQUAL Linux)
     endif()
     set(CLR_CMAKE_PLATFORM_LINUX 1)
 
-    # Detect Alpine Linux
-    SET(OS_RELEASE_FILENAME "/etc/os-release")
+    # Detect Alpine & Tizen Linux
+    if(DEFINED ENV{CROSSCOMPILE})
```

Comment:
I prefer not to use env vars as switches controlling cmake scripts. Especially here where we alredy have a similar switch passed by CLR_CROSS_COMPONENTS_BUILD cmake variable. 
Could you please change it to use CLR_CROSSCOMPILE cmake variable and set / pass it to cmake in `gen-buildsys-clang.sh` similarly to how we set / pass the CLR_CROSS_COMPONENTS_BUILD?